recipes/simbody/bld.bat

```diff
@@ -0,0 +1,8 @@
+mkdir build
+cd build
+cmake -G "%CMAKE_GENERATOR%" -DBUILD_VISUALIZER=OFF -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%"
+cmake --build . --target install --config Release -- /verbosity:quiet
+REM NOTE: Run just one test here in the build directory to make sure things
+REM built correctly. This cannot be specified in the meta.yml:test section
+REM because it won't be run in the build directory.
```

Comment:
I'm not quite sure, but the build directory may not exist at that point. I can try though.